orttraining/orttraining/training_ops/cpu/torch/torch_custom_function_kernel_base.cc

```diff
@@ -84,13 +84,6 @@ void PythonOpBase::Init(const OpKernelInfo& info) {
   CreateArgPositions();
 }
 
-void PythonOpBase::Clear() {
-  for (auto ptr : const_args_) {
-    auto obj = reinterpret_cast<PyObject*>(ptr);
-    Py_DECREF(obj);
```

Comment:
think twice, remove all the clean up code is not correct. We should only remove the clean up for pointer-type scalars.

Reply:
will make a change...